File: a_system/src/main/java/com/example/a_system/vo/ChoiceVO.java
package com.example.a_system.vo;

import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;

@JacksonXmlRootElement(localName = "A系统学生选课信息")
public class ChoiceVO {
    private String cno;
    private StudentVO student;
    private String grade;

    public ChoiceVO(String cno, StudentVO student, String grade) {
        this.cno = cno;
        this.student = student;
        this.grade = grade;
    }

    @JacksonXmlProperty(localName = "课程编号A")
    public String getCno() {
        return cno;
    }

    public void setCno(String cno) {
        this.cno = cno;
    }

    @JacksonXmlProperty(localName = "学生信息A")
    public StudentVO getStudent() {
        return student;
    }

    public void setStudent(StudentVO student) {
        this.student = student;
    }

    @JacksonXmlProperty(localName = "学生成绩A")
    public String getGrade() {
        return grade;
    }

    public void setGrade(String grade) {
        this.grade = grade;
    }
}

File: system_b/educationalSystem_B/vo/choiceVO.py

class ChoiceVO:
    def __init__(self, cno, student, grade):
        self.cno = cno
        self.student = student
        self.grade = grade

    def getCno(self):
        return self.cno

    def setCno(self, cno):
        self.cno = cno

    def getStudent(self):
        return self.student

    def setStudent(self, student):
        self.student = student

    def getGrade(self):
        return self.grade

    def setGrade(self, grade):
        self.grade = grade